static/js/twitter.js

```diff
@@ -8,5 +8,40 @@
   js = d.createElement(s)
   js.id = id
   js.src = p + '://platform.twitter.com/widgets.js'
+  js.setAttribute('load', 'twitter_filter()')
   fjs.parentNode.insertBefore(js, fjs)
 })(document, 'script', 'twitter-wjs')
+
+function twitter_filter() {
+  const node = obj => obj.parentNode.parentNode.parentNode.parentNode.parentNode
+  var twitter_prefix = 20
+  var twitter_handles = []
+  var orgs = document.getElementsByClassName('org-info')
+  for (var i = 0; i < orgs.length; i++) {
+    var org_info = orgs[i]
+    for (var j = 0; j < org_info.children.length; j++) {
+      if (org_info.children[j].href) {
+        if (org_info.children[j].href.startsWith('https://twitter.com/')) {
+          var twitter_username = org_info.children[j].href.substr(
+            twitter_prefix
+          )
+          twitter_handles.push(twitter_username)
+        }
+      }
+    }
+  }
+  if (document.getElementById('twitter-widget-0') !== null) {
+    var tweet_authors = document
+      .getElementById('twitter-widget-0')
+      .contentWindow.document.getElementsByClassName(
+        'TweetAuthor-screenName Identity-screenName'
+      )
+    for (var k = 0; k < tweet_authors.length; k++) {
+      if (twitter_handles.indexOf(tweet_authors[k].innerHTML.substr(1)) > -1) {
```

Comment:
There is still no difference with the current Twitter feed. Some feeds still repeated.